src/main.rs

```diff
@@ -157,7 +157,7 @@ async fn main() -> IoResult<()> {
         App::new()
             .app_data(Data::clone(&config))
             .app_data(Data::new(http_client))
-            .wrap(Logger::default())
+            .wrap(Logger::new("%{r}a \"%r\" %s %b \"%{Referer}i\" \"%{User-Agent}i\" %T"))
```

Comment:
Can you share an example of how this log entry looks like?

Reply:
Also, what was the reason behind that change?

Reply:
If my IP address is `1.2.3.4` and I am using a reverse proxy (as I mentioned in https://github.com/orhun/rustypaste/issues/54#issuecomment-1586487208), the log file will show the following:

```
[2023-06-12T02:22:15Z INFO  actix_web::middleware::logger] 127.0.0.1 "GET / HTTP/1.1" 200 271 "-" "Mozilla/5.0 Gecko/20100101 Firefox/114.0" 0.000036
```

With this PR, the log will show:

```
[2023-06-12T02:22:15Z INFO  actix_web::middleware::logger] 1.2.3.4 "GET / HTTP/1.1" 200 271 "-" "Mozilla/5.0 Gecko/20100101 Firefox/114.0" 0.000036`
```

Reply:
Got it. Just checked the [default logger format](https://docs.rs/actix-web/latest/actix_web/middleware/struct.Logger.html#method.default) and it makes sense!